recipes/catch2/2.x.x/conanfile.py

```diff
@@ -35,27 +35,48 @@ def _configure_cmake(self):
         self._cmake.definitions["BUILD_TESTING"] = "OFF"
         self._cmake.definitions["CATCH_INSTALL_DOCS"] = "OFF"
         self._cmake.definitions["CATCH_INSTALL_HELPERS"] = "ON"
-        self._cmake.configure(
-            source_folder=self._source_subfolder,
-            build_folder=self._build_subfolder
-        )
+        self._cmake.configure(build_folder=self._build_subfolder)
         return self._cmake
 
+    def build(self):
+        # Catch2 does skip install if included as subproject:
+        # https://github.com/catchorg/Catch2/blob/79a5cd795c387e2da58c13e9dcbfd9ea7a2cfb30/CMakeLists.txt#L100-L102
+        main_cml = os.path.join(self._source_subfolder, "CMakeLists.txt")
+        tools.replace_in_file(main_cml, "if (NOT_SUBPROJECT)", "if (TRUE)")
+        if self.version >= "2.13.4":
+            cmake = self._configure_cmake()
+            cmake.build()
+        else:
+            no_copy_source = True
```

Comment:
I do not know if this has any effect here?

Reply:
doubtful, that causes the build method to be skipped... but it's in the build method 🤔

Reply:
So should I delete the else branch? @uilianries any feedback from you on that?

Reply:
I deleted it, looks like its not required at all